includes/pegasus-list-functions.php

```diff
@@ -53,7 +53,7 @@ function today_sidebar_pegasus_list_content( $content, $items, $args, $fallback_
 		</div>
 
 		<div class="ucf-pegasus-list-featured-story position-relative d-flex flex-column mb-5">
-			<p class="mb-2 text-muted text-uppercase small">Featured Story</p>
+			<span class="mb-2 text-muted text-uppercase small">Featured Story</span>
```

Comment:
We proabbly want to add `d-block` to this `span` to keep it as a block-level element and ensure the margin adjustments behave consistently cross-browser.

Reply:
Actually, closing this--didn't realize this was wrapped in a flex container, so the p/span are already inheriting flex styles anyway.